pkg/scheduler/internal/cache/cache.go

```diff
@@ -90,15 +90,15 @@ type imageState struct {
 }
 
 // createImageStateSummary returns a summarizing snapshot of the given image's state.
-func (cache *schedulerCache) createImageStateSummary(state *imageState) *framework.ImageStateSummary {
+func (cache *cacheImpl) createImageStateSummary(state *imageState) *framework.ImageStateSummary {
 	return &framework.ImageStateSummary{
 		Size:     state.size,
 		NumNodes: len(state.nodes),
 	}
 }
 
-func newSchedulerCache(ttl, period time.Duration, stop <-chan struct{}) *schedulerCache {
-	return &schedulerCache{
+func newSchedulerCache(ttl, period time.Duration, stop <-chan struct{}) *cacheImpl {
```

Comment:
`newSchedulerCache` --> `newCacheImpl` or just `newCache`?

Reply:
At first, I renamed this to `newCacheImpl`  , but then I realized it's just one implementation, so I changed back. `newCache` seems good to me.